### PR TITLE
feat: add space filter for org level app installations

### PR DESCRIPTION
**Pull request by damienxy**
## Summary

This PR adds an optional space filter to the `appInstallation.getForOrganisation` method.

## Checklist (check all before merging)

- [ ] Both unit and integration tests are passing
- [ ] There are no breaking changes
- [ ] Changes are reflected in the documentation

When adding a new method:

- [ ] The new method is exported through the default and plain CMA client
- [ ] All new public types are exported from `./lib/export-types.ts`
- [ ] Added a unit test for the new method
- [ ] Added an integration test for the new method
- [ ] The new method is added to the documentation

### CODE REVIEW

**Review by mayakarabula — APPROVED** [review]

File: lib/common-types.ts [hunk 1]
```diff
@@ -714,7 +718,7 @@ export type MRActions = {
     }
     delete: { params: GetAppInstallationParams; return: any }
     getForOrganization: {
-      params: GetOrganizationParams & { appDefinitionId: string }
+      params: GetOrganizationParams & { appDefinitionId: string; spaceId?: string }
```
Comment (marcolink) [inline lib/common-types.ts:721]: I'm wondering if the space id belongs to a query object 🤔 ?

Comment (marcolink) [reply lib/common-types.ts:721]: It was just a question - especially thinking about extending it in the future with features like pagination.

Comment (marcolink) [reply lib/common-types.ts:721]: Wondering if we can document somehow that this is a "filter"?

Comment (damienxy) [reply lib/common-types.ts:721]: > Wondering if we can document somehow that this is a "filter"?

How & where would you like to document it?

Comment (damienxy) [reply lib/common-types.ts:725]: Update: There was an issue due to default spaces and I had to put it in a query object after all, also adjusted the JSDoc.

**Review by marcolink — APPROVED** [review]

**Pull request merged**